airbyte-connector-builder/build.gradle

```diff
@@ -0,0 +1,25 @@
+import org.openapitools.generator.gradle.plugin.tasks.GenerateTask
+
+plugins {
+    id "org.openapi.generator" version "5.3.1"
+    id 'airbyte-python'
+    id 'airbyte-docker'
+}
+
+airbytePython {
+    moduleDirectory 'connector_builder'
+}
+
+
+task generateApiClient(type: GenerateTask) {
+    inputSpec = "$rootDir.absolutePath/airbyte-api/src/main/openapi/config.yaml"
+    outputDir = "$buildDir/airbyte_api_client"
```

Comment:
I think these should be changed to take in the connector builder openAPI spec as input, and output something named accordingly (this should be generating a server rather than a client?)

Though, this is covered by the ticket that Sherif was planning to work on, so maybe not necessary to do in this PR

Reply:
We may also want to move the connector builder server OpenAPI spec into this new `airbyte-connector-builder` module, since currently it lives in this `connector-builder-server` directory all on its own: https://github.com/airbytehq/airbyte/tree/master/connector-builder-server/src/main/openapi

Reply:
[done](https://github.com/airbytehq/airbyte/pull/18410/commits/57dda047235d386e5a5a781288541699db1ab73a)